controllers/vspheremachine_annotations.go

```diff
@@ -1,74 +0,0 @@
-/*
```

Comment:
Can we remove this file in a separate commit?

Reply:
Sure. PR now contains 2 commits.